projects/storefrontapp-e2e-cypress/cypress/helpers/navigation.ts

```diff
@@ -5,15 +5,20 @@ export const navigation = {
   }: {
     queryStringParams?: string;
     options?: Partial<Cypress.VisitOptions>;
-  }): void {
+  }): Cypress.Chainable<Window> {
     queryStringParams =
       !queryStringParams || queryStringParams.indexOf('?') !== -1
         ? queryStringParams
         : `?${queryStringParams}`;
-    cy.visit(`/${queryStringParams ? queryStringParams : ''}`, options);
+    return cy.visit(`/${queryStringParams ? queryStringParams : ''}`, options);
   },
-  goToProduct(id): void {
-    cy.visit(`/product/${id}`);
+  goToProduct(id): Cypress.Chainable<Window> {
```

Comment:
wondering if there was a reason for this change and same for above

Reply:
I don't mind keeping it like this but just want to know

Reply:
@bgambocjaviniar for this and also the waitforpage, this was for the case that I wanted to chain them. When I was writing the tests at one point I needed the chaining, but I no longer do. I can't remember the exact reason.